### ISSUE TITLE
Change "BuildTime" into a "build_time" AbilitySpecial

### ISSUE THREAD

**Issue by MNoya** [stephenfournier/Dota-2-Building-Helper#21]
In preparation for tooltips, I think every building ability should have this block:

~~~
"AbilitySpecial"
{
	"01"
	{
		"var_type"				"FIELD_FLOAT"
		"build_time"			"60.0"
	}
}
~~~

And "BuildTime"	"60.0" could be removed from the required // BuildingHelper info, always checking the special value instead.

**Comment by Myll** [comment]
Good idea. Let me know if more things should be AbilitySpecials.

**Comment by MNoya** [comment]
After reviewing this, I think it's not worth it, kinda hard to parse and not flexible
